docs/data/preprocessing_data.rst

```diff
@@ -30,3 +30,82 @@ cf-python:
 
     # Write the combined data to a single file
     cf.write(field, "combined-output.nc")
+
+Combine Variables
+-----------------
+
+Variables will often be stored in separate files. To combine them with cf-python simply
+read them in separately and then write them together:
+
+.. code-block:: python
+
+    # Read the separate input files
+    field1 = cf.read("var1_file.nc")[0]
+    field2 = cf.read("var2_file.nc")[0]
+
+    # Write the combined fields to a single file
+    cf.write([field1, field2], "combined_file.nc")
+
+Regridding
+----------
```

Comment:
Might be worth adding a note about how regridding is done (linear, nearest neighbour, other) and pointing to the CF/ESMF API docs for more info?